src/components/category.js

```diff
@@ -1,4 +1,6 @@
-const Category = () => (
-  <section className="home" />
-);
+import React from 'react';
+
+function Category() {
+  return <div>under construction</div>;
+}
```

Comment:
- [ ] Kindly, make sure that you implemented a `Check status` button in your categories page and  that button should dispatch the `underConstruction` action creator. Means, once we click on that `Check status` button we should see the text "Under Construction".

- [ ] Also, kindly remove the form to add a new book from your categories page.
![image](https://user-images.githubusercontent.com/71874425/179365378-86eba804-a9ee-4274-9f56-71521c718e15.png)